app/assets/styles/app-imports.scss

```diff
@@ -88,3 +88,5 @@
 @import "../../modules/onboarding/assets/js/pages/site-logo";
 @import "../../modules/onboarding/assets/js/pages/good-to-go";
 @import "../../modules/onboarding/assets/js/pages/upload-and-install-pro";
+@import "../../../modules/announcements/assets/js/components/Overlay/Overlay.css";
+@import "../../../modules/announcements/assets/js/components/Overlay.css";
```

Comment:
Why we need this here ?
Also as you can see there is a standard here. don't use Capital in the name.